MaintenanceManager/MaintenanceManager.cpp

```diff
@@ -245,6 +245,9 @@ namespace WPEFramework {
             :PluginHost::JSONRPC()
         {
             MaintenanceManager::_instance = this;
+#ifdef DISABLE_THUNDER_CLIENT_IN_MAINTENANCE_MANAGER_GTEST
```

Comment:
We should not make any changes to source code to support unit testing. If you want a null ptr, pls add a mock in unit test code. You can refer to other unit tests for reference.